pandas/_libs/src/headers/cmath

```diff
@@ -0,0 +1,13 @@
+#ifndef _PANDAS_MATH_H_
+#define _PANDAS_MATH_H_
+
+#if defined(_MSC_VER) && (_MSC_VER < 1800)
+#include <cmath>
+namespace std {
+  __inline int signbit(double num) { return _copysign(1.0, num) < 0; }
```

Comment:
I would revert this to use `math.h` - technically extending namespace `std` is undefined behavior so you'd need to do it a different way